aten/src/ATen/native/native_functions.yaml

```diff
@@ -2701,16 +2701,29 @@
 
 - func: matrix_exp_backward(Tensor self, Tensor grad) -> Tensor
 
+# DEPRECATED: Use torch.aminmax instead
 - func: _aminmax(Tensor self) -> (Tensor, Tensor)
   variants: function
   dispatch:
     CPU, CUDA: _aminmax_all
 
+# DEPRECATED: Use torch.aminmax instead
 - func: _aminmax.dim(Tensor self, int dim, bool keepdim=False) -> (Tensor, Tensor)
   variants: function
   dispatch:
     CPU, CUDA: _aminmax
 
+- func: aminmax(Tensor self, *, int? dim=None, bool keepdim=False) -> (Tensor min, Tensor max)
+  device_check: NoCheck
```

Comment:
Inline comment for why this check is skipped like here:

https://github.com/pytorch/pytorch/blob/725d98bab61d24509ffc199cee6d177f32a96e38/aten/src/ATen/native/native_functions.yaml#L208